docs/docker_run.rst

```diff
@@ -0,0 +1,190 @@
+docker_run
+==========
+
+girder_worker provides a built-in task that can be used to run docker containers.
+girder_worker makes it easy to work on data held in girder from with a
+docker containers.
+
+Container arguments
+-------------------
+
+The ``docker_run`` task exposes a ``container_args`` parameter which can be using
+to pass arguments to the container entrypoint.
+
+
+Volumes
+-------
+
+The volumes to be mount into a container can me pass to the ``docker_run`` task
+in one of two ways.
+
+Using docker-py syntax
+^^^^^^^^^^^^^^^^^^^^^^
+
+In this case the value of the ``volumes`` parameter is a ``dict`` conforming to
+specification defined by `docker-py <http://docker-py.readthedocs.io/en/stable/containers.html>`_,
+this is passed directly to docker-py. For example
+
+.. code-block:: python
+
+    volumes = {
+         '/home/docker/data': {
+             'bind': '/mnt/docker/',
+             'mode': 'rw'
+         }
+
+    }
+    docker_run.delay('my/image', pull_image=True, volumes=volumes)
+
+Using the Volume class
+^^^^^^^^^^^^^^^^^^^^^^
+
+girder_worker provides a utility class ``Volume`` that can be using to define volumes
```

Comment:
Let's either provide the full module path to this class, or even better, link to its API docs.